File: frontend/src/models/request/EvaluationPeriodRequest.js
export class EvaluationPeriodRequest {
    constructor(name, startDate, endDate, templateIds, isSelfEvaluationIncluded) {
        this.name = name;
        this.startDate = startDate;
        this.endDate = endDate;
        this.templateIds = templateIds;
        this.isSelfEvaluationIncluded = isSelfEvaluationIncluded;
    }
}

File: backend/src/main/java/com/batuhan/feedback360/model/request/EvaluationPeriodRequest.java
package com.batuhan.feedback360.model.request;

import com.batuhan.feedback360.model.enums.PeriodStatus;
import java.time.LocalDateTime;
import lombok.Data;

@Data
public class EvaluationPeriodRequest {
    private String periodName;
    private String internalPeriodName;
    private String evaluationName;
    private LocalDateTime startDate;
    private LocalDateTime endDate;
    private PeriodStatus status;
}
